tools/webpack/vendors.js

```diff
@@ -1,39 +1,121 @@
 /**
  * External dependencies
  */
-const { join } = require( 'path' );
+const { join, basename } = require( 'path' );
+const { get, isString } = require( 'lodash' );
+const webpack = require( 'webpack' );
+
+/**
+ * WordPress dependencies
+ */
+const CustomTemplatedPathPlugin = require( '@wordpress/custom-templated-path-webpack-plugin' );
 
 const baseDir = join( __dirname, '../../' );
 
 module.exports = function( env = { environment: 'production', watch: false } ) {
 	const mode = env.environment;
 	const suffix = mode === 'production' ? '.min': '';
 
-	const vendors = {
-		'lodash': 'lodash',
-		'wp-polyfill': '@babel/polyfill',
-		'moment': 'moment',
-		'react': 'react',
-		'react-dom': 'react-dom',
-	};
+	const vendors = [
+		'lodash',
+		{
+			packageName: '@babel/polyfill',
+			global: 'wp-polyfill',
+			filename: 'wp-polyfill',
+		},
+		{
+			packageName: 'whatwg-fetch',
+			global: 'wp-polyfill-fetch',
+			filename: 'wp-polyfill-fetch',
+		},
+		{
+			packageName: 'element-closest',
+			global: 'wp-polyfill-element-closest',
+			filename: 'wp-polyfill-element-closest',
+		},
+		{
+			packageName: 'polyfill-library/polyfills/Node/prototype/contains/polyfill.js',
+			global: 'wp-polyfill-node-contains',
+			filename: 'wp-polyfill-node-contains',
+		},
+		{
+			packageName: 'formdata-polyfill',
+			global: 'wp-polyfill-formdata',
+			filename: 'wp-polyfill-formdata',
+		},
+		'moment',
+		{
+			packageName: 'react',
+			global: 'React',
+		},
+		{
+			packageName: 'react-dom',
+			global: 'ReactDOM',
+		},
```

Comment:
I feel some of these packages don't need globals. Is it concerning that we always attach a global?

Reply:
More globally, I wonder if we should use webpack to include these scripts or just "copy" task to copy from node_modules into a given directory in Core. It feels like all these packages already have built files with globals etc...

Reply:
I checked all the vendors and they all already have build versions, so I changed this to use the Webpack copy plugin: https://github.com/Yoast/wordpress-develop-mirror/pull/118/commits/25dcd5e674aadc2167bfd1db82722e621e16d4fb. 3 packages didn't have minified files, so I used uglify to minify those.